docs/jwst/ramp_fitting/main.rst

```diff
@@ -37,45 +37,50 @@ a given pixel, the final slope is determined as a weighted average from all
 intervals and is written to a file as the primary output product.  In this
 output product, the 4-D GROUPDQ from all integrations is compressed into 2-D,
 which is then merged (using a bitwise OR) with the input 2-D PIXELDQ to create
-the output PIXELDQ.  If the ramp fitting step is run by itself, the output file 
-name will have the suffix '_RampFit' or the suffix '_RampFitStep'; if the ramp 
-fitting step is run as the final step in a pipeline, the final output file name 
-will have the suffix '_rate'. In either case, the user can override this name by 
-specifying an output file name.
+the output PIXELDQ.  The 3-D VAR_POISSON and VAR_RNOISE arrays from all 
+integrations are averaged into corresponding 2-D output arrays.  If the ramp 
+fitting step is run by itself, the output file name will have the suffix 
+'_RampFit' or the suffix '_RampFitStep'; if the ramp fitting step is run as the 
+final step in a pipeline, the final output file name will have the suffix '_rate'.
```

Comment:
Technically ramp_fit is no longer the final step in the level-2a (calwebb_detector1) pipeline, so perhaps it's best to just say "run as part of the `calwebb_detector1` pipeline ..."